README.md

```diff
@@ -1,2 +1,23 @@
-finn-no.github.com
-==================
+# finn-no.github.com
+
+Maintainers:
+
+- @nicolaihoge (primary maintainer)
```

Comment:
@nicolaihoge are you comfortable with being the primary maintainer (and thus point of contact) for the FINN open source website?